server/package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Backend subproject for offline-markdown-previewer",
   "main": "server.js",
   "config": {
-    "port": "8080"
+    "http_port": "8080",
+    "ws_port": "8081"
```

Comment:
No strong feelings one way or another, one thing though - if there's a config parameter used to start server, it should be used on the client as well, otherwise, somebody, who overrides this port, will get an error